lib/msal-core/src/Utils.ts

```diff
@@ -406,4 +407,21 @@ export class Utils {
 
     return url.indexOf(suffix, url.length - suffix.length) !== -1;
   }
+
+     static checkSSO(extraQueryParameters: string)
+    {
+        return  !(extraQueryParameters &&  ((extraQueryParameters.indexOf(Constants.login_hint) !== -1 ||  extraQueryParameters.indexOf(Constants.sid) !== -1 )));
+    }
+
+     static constructUnifiedCacheExtraQueryParameter(idTokenObject : any, extraQueryParameters ?: string)
+    {
+        if(idTokenObject && idTokenObject.upn) {
+            return  extraQueryParameters = "&" + Constants.login_hint+ "=" + idTokenObject.upn + "&" + Constants.domain_hint + "=" + Constants.organizations;
+        }
+        else if(idTokenObject &&  this.isEmpty(idTokenObject.upn)){
```

Comment:
I don't think we need to call isEmpty on the idTokenObject.upn . We can just check to see if idTokenObject has the property upn.

Reply:
when we spoke to server team today I think they mentioned it would be empty/null. I will confirm and close this.